starport/services/network/networkchain/init.go

```diff
@@ -118,3 +123,22 @@ func (c *Chain) checkGenesis(ctx context.Context) error {
 	// example: gentxs formats are not checked
 	// to perform a full validity check of the genesis we must try to start the chain with sample accounts
 }
+
+// checkGentxFiles checks if a gentx exist for the chain
+func (c *Chain) checkGentxFiles() error {
+	gentxPath, err := c.DefaultGentxPath()
```

Comment:
Do we need to check for the default, single gentx path first? We already have the check for dir starting from line 133.